specification/network/resource-manager/Microsoft.Network/stable/2020-08-01/examples/VirtualNetworkGatewayConnectionGetIkeSas.json

```diff
@@ -0,0 +1,14 @@
+{
+    "parameters": {
+        "api-version": "2020-08-01",
+        "subscriptionId": "subid",
+        "resourceGroupName": "rg1",
+        "virtualNetworkGatewayConnectionName": "vpngwcn1"
+    },
+    "responses": {
+        "202": {},
```

Comment:
Can you please add a location header for example?

Reply:
Done